client/components/Form/PasswordReveal/PasswordReveal.tsx

```diff
@@ -34,6 +34,9 @@ function PasswordReveal({
           className="absolute text-grey"
           onClick={() => setVisible(!visible)}
           title={visible ? 'Hide password' : 'Show password'}
+          style={{
```

Comment:
It's not a good practice to use inline styles or negative margins. 

Have a look in `PasswordReveal.module.scss` and add modify the `top` property to have it centered.